ci/publish-artifactory.sh

```diff
@@ -28,11 +28,14 @@ push() {
 
 TRIGGER_RUNNER=daml-trigger-runner-$RELEASE_TAG.jar
 SCRIPT_RUNNER=daml-script-$RELEASE_TAG.jar
+NON_REPUDIATION_APP=non-repudiation-app-$RELEASE_TAG.jar
```

Comment:
do we want to include `app` in there? We don’t do that for anything else and this doesn’t seem any more or less of an app than other components.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9238/commits/9cca0c2e67c0e73824e0fcec247196025c0e8f8b